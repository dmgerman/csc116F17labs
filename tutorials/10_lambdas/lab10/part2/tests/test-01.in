6 10 17 -1
Apple Raspberry Pear
