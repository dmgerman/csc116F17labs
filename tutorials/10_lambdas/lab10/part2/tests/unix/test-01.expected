Enter a sequence of integers, ending with -1: 
Vector VI: 6 10 17 
Enter a sequence of words: 
Vector VS: Apple Raspberry Pear 

Negation of VI: -6 -10 -17 

VI + 6: 12 16 23 

Lengths of elements of VS: 5 9 4 

VS (doubled): AppleApple RaspberryRaspberry PearPear 

Template tests: 
Negation of VI: -6 -10 -17 

VI + 6: 12 16 23 

Lengths of elements of VS: 5 9 4 

VS (doubled): AppleApple RaspberryRaspberry PearPear 

